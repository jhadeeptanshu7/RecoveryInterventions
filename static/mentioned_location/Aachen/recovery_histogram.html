



<!DOCTYPE html>
<html lang="en">
  
  <head>
    
      <meta charset="utf-8">
      <title>Bokeh Plot</title>
      
      
        
          
        <link rel="stylesheet" href="https://cdn.pydata.org/bokeh/release/bokeh-1.0.1.min.css" type="text/css" />
        
        
          
        <script type="text/javascript" src="https://cdn.pydata.org/bokeh/release/bokeh-1.0.1.min.js"></script>
        <script type="text/javascript">
            Bokeh.set_log_level("info");
        </script>
        
      
      
    
  </head>
  
  
  <body>
    
      
        
          
          
            
              <div class="bk-root" id="8f01db74-db65-4909-bd9a-867db53fe04e"></div>
            
          
        
      
      
        <script type="application/json" id="190840">
          {"3f9b3a50-7c0a-493d-b474-c85117dcfe4f":{"roots":{"references":[{"attributes":{},"id":"190757","type":"WheelZoomTool"},{"attributes":{"bottom_units":"screen","fill_alpha":{"value":0.5},"fill_color":{"value":"lightgrey"},"left_units":"screen","level":"overlay","line_alpha":{"value":1.0},"line_color":{"value":"black"},"line_dash":[4,4],"line_width":{"value":2},"plot":null,"render_mode":"css","right_units":"screen","top_units":"screen"},"id":"190764","type":"BoxAnnotation"},{"attributes":{},"id":"190761","type":"HelpTool"},{"attributes":{},"id":"190760","type":"ResetTool"},{"attributes":{},"id":"190748","type":"CategoricalTicker"},{"attributes":{"axis_label":"Recovery Terms","formatter":{"id":"190778","type":"CategoricalTickFormatter"},"major_label_orientation":"vertical","plot":{"id":"190737","subtype":"Figure","type":"Plot"},"ticker":{"id":"190748","type":"CategoricalTicker"}},"id":"190747","type":"CategoricalAxis"},{"attributes":{},"id":"190745","type":"LinearScale"},{"attributes":{},"id":"190743","type":"CategoricalScale"},{"attributes":{"callback":null,"start":0},"id":"190741","type":"DataRange1d"},{"attributes":{"grid_line_color":{"value":null},"plot":{"id":"190737","subtype":"Figure","type":"Plot"},"ticker":{"id":"190748","type":"CategoricalTicker"}},"id":"190750","type":"Grid"},{"attributes":{"axis_label":"Number of Drug Users","formatter":{"id":"190780","type":"BasicTickFormatter"},"plot":{"id":"190737","subtype":"Figure","type":"Plot"},"ticker":{"id":"190752","type":"BasicTicker"}},"id":"190751","type":"LinearAxis"},{"attributes":{"data_source":{"id":"190771","type":"ColumnDataSource"},"glyph":{"id":"190772","type":"VBar"},"hover_glyph":null,"muted_glyph":null,"nonselection_glyph":{"id":"190773","type":"VBar"},"selection_glyph":null,"view":{"id":"190775","type":"CDSView"}},"id":"190774","type":"GlyphRenderer"},{"attributes":{"source":{"id":"190771","type":"ColumnDataSource"}},"id":"190775","type":"CDSView"},{"attributes":{"fill_color":{"value":"#1f77b4"},"line_color":{"value":"#1f77b4"},"top":{"field":"top"},"width":{"value":0.5},"x":{"field":"x"}},"id":"190772","type":"VBar"},{"attributes":{"fill_alpha":{"value":0.1},"fill_color":{"value":"#1f77b4"},"line_alpha":{"value":0.1},"line_color":{"value":"#1f77b4"},"top":{"field":"top"},"width":{"value":0.5},"x":{"field":"x"}},"id":"190773","type":"VBar"},{"attributes":{},"id":"190756","type":"PanTool"},{"attributes":{"callback":null,"data":{"top":[2,1,1,1,1,1,1,1,1,1,1,1,1,1,1,1,1,1,1,1,1,1,1,1,1,1,1,1,1,1,1,1,1,1,1,1,1],"x":["groups","facility","alive","therapist","asleep","sober","depression","death","court","binge","father","treatment","frustration","prison","territory","venture","smart","life","pain","withdraw","degree","opiates","withdrawals","abuse","effects","nausea","anxiety","programs","university","hungry","journey","jail","methadone","mother","clean","chills","medical"]},"selected":{"id":"190781","type":"Selection"},"selection_policy":{"id":"190782","type":"UnionRenderers"}},"id":"190771","type":"ColumnDataSource"},{"attributes":{"overlay":{"id":"190764","type":"BoxAnnotation"}},"id":"190758","type":"BoxZoomTool"},{"attributes":{},"id":"190759","type":"SaveTool"},{"attributes":{"plot":null,"text":"Recovery Term Counts"},"id":"190736","type":"Title"},{"attributes":{"below":[{"id":"190747","type":"CategoricalAxis"}],"left":[{"id":"190751","type":"LinearAxis"}],"plot_height":300,"plot_width":2000,"renderers":[{"id":"190747","type":"CategoricalAxis"},{"id":"190750","type":"Grid"},{"id":"190751","type":"LinearAxis"},{"id":"190755","type":"Grid"},{"id":"190764","type":"BoxAnnotation"},{"id":"190774","type":"GlyphRenderer"}],"title":{"id":"190736","type":"Title"},"toolbar":{"id":"190762","type":"Toolbar"},"x_range":{"id":"190739","type":"FactorRange"},"x_scale":{"id":"190743","type":"CategoricalScale"},"y_range":{"id":"190741","type":"DataRange1d"},"y_scale":{"id":"190745","type":"LinearScale"}},"id":"190737","subtype":"Figure","type":"Plot"},{"attributes":{},"id":"190778","type":"CategoricalTickFormatter"},{"attributes":{},"id":"190752","type":"BasicTicker"},{"attributes":{"callback":null,"factors":["groups","facility","alive","therapist","asleep","sober","depression","death","court","binge","father","treatment","frustration","prison","territory","venture","smart","life","pain","withdraw","degree","opiates","withdrawals","abuse","effects","nausea","anxiety","programs","university","hungry","journey","jail","methadone","mother","clean","chills","medical"],"range_padding":0.01},"id":"190739","type":"FactorRange"},{"attributes":{"active_drag":"auto","active_inspect":"auto","active_multi":null,"active_scroll":"auto","active_tap":"auto","tools":[{"id":"190756","type":"PanTool"},{"id":"190757","type":"WheelZoomTool"},{"id":"190758","type":"BoxZoomTool"},{"id":"190759","type":"SaveTool"},{"id":"190760","type":"ResetTool"},{"id":"190761","type":"HelpTool"}]},"id":"190762","type":"Toolbar"},{"attributes":{},"id":"190782","type":"UnionRenderers"},{"attributes":{},"id":"190781","type":"Selection"},{"attributes":{},"id":"190780","type":"BasicTickFormatter"},{"attributes":{"dimension":1,"plot":{"id":"190737","subtype":"Figure","type":"Plot"},"ticker":{"id":"190752","type":"BasicTicker"}},"id":"190755","type":"Grid"}],"root_ids":["190737"]},"title":"Bokeh Application","version":"1.0.1"}}
        </script>
        <script type="text/javascript">
          (function() {
            var fn = function() {
              Bokeh.safely(function() {
                (function(root) {
                  function embed_document(root) {
                    
                  var docs_json = document.getElementById('190840').textContent;
                  var render_items = [{"docid":"3f9b3a50-7c0a-493d-b474-c85117dcfe4f","roots":{"190737":"8f01db74-db65-4909-bd9a-867db53fe04e"}}];
                  root.Bokeh.embed.embed_items(docs_json, render_items);
                
                  }
                  if (root.Bokeh !== undefined) {
                    embed_document(root);
                  } else {
                    var attempts = 0;
                    var timer = setInterval(function(root) {
                      if (root.Bokeh !== undefined) {
                        embed_document(root);
                        clearInterval(timer);
                      }
                      attempts++;
                      if (attempts > 100) {
                        console.log("Bokeh: ERROR: Unable to run BokehJS code because BokehJS library is missing");
                        clearInterval(timer);
                      }
                    }, 10, root)
                  }
                })(window);
              });
            };
            if (document.readyState != "loading") fn();
            else document.addEventListener("DOMContentLoaded", fn);
          })();
        </script>
    
  </body>
  
</html>