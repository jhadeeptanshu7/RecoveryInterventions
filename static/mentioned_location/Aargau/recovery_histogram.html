



<!DOCTYPE html>
<html lang="en">
  
  <head>
    
      <meta charset="utf-8">
      <title>Bokeh Plot</title>
      
      
        
          
        <link rel="stylesheet" href="https://cdn.pydata.org/bokeh/release/bokeh-1.0.1.min.css" type="text/css" />
        
        
          
        <script type="text/javascript" src="https://cdn.pydata.org/bokeh/release/bokeh-1.0.1.min.js"></script>
        <script type="text/javascript">
            Bokeh.set_log_level("info");
        </script>
        
      
      
    
  </head>
  
  
  <body>
    
      
        
          
          
            
              <div class="bk-root" id="9b1a2c58-0eeb-4f38-bf69-a1cec48bd683"></div>
            
          
        
      
      
        <script type="application/json" id="66192">
          {"9ff20b4d-f3f7-446d-87bc-db9dce11ee39":{"roots":{"references":[{"attributes":{"callback":null,"data":{"top":[2,2,2,1,1,1,1,1,1,1,1,1,1,1,1,1,1,1,1,1,1,1,1,1,1,1,1,1,1,1,1,1,1,1,1],"x":["life","effects","medical","medication","lethargy","benzo","asleep","counseling","depression","relapse","divorce","binge","sexuality","hydrated","quitting","comedown","opiates","mother","insomnia","withdrawals","discomfort","therapy","groups","dt","pain","recovery","anxiety","soreness","dependence","dependency","benzos","cravings","withdraw","withdrawal","addiction"]},"selected":{"id":"66133","type":"Selection"},"selection_policy":{"id":"66134","type":"UnionRenderers"}},"id":"66123","type":"ColumnDataSource"},{"attributes":{"data_source":{"id":"66123","type":"ColumnDataSource"},"glyph":{"id":"66124","type":"VBar"},"hover_glyph":null,"muted_glyph":null,"nonselection_glyph":{"id":"66125","type":"VBar"},"selection_glyph":null,"view":{"id":"66127","type":"CDSView"}},"id":"66126","type":"GlyphRenderer"},{"attributes":{"source":{"id":"66123","type":"ColumnDataSource"}},"id":"66127","type":"CDSView"},{"attributes":{"fill_color":{"value":"#1f77b4"},"line_color":{"value":"#1f77b4"},"top":{"field":"top"},"width":{"value":0.5},"x":{"field":"x"}},"id":"66124","type":"VBar"},{"attributes":{"fill_alpha":{"value":0.1},"fill_color":{"value":"#1f77b4"},"line_alpha":{"value":0.1},"line_color":{"value":"#1f77b4"},"top":{"field":"top"},"width":{"value":0.5},"x":{"field":"x"}},"id":"66125","type":"VBar"},{"attributes":{"callback":null,"start":0},"id":"66093","type":"DataRange1d"},{"attributes":{"callback":null,"factors":["life","effects","medical","medication","lethargy","benzo","asleep","counseling","depression","relapse","divorce","binge","sexuality","hydrated","quitting","comedown","opiates","mother","insomnia","withdrawals","discomfort","therapy","groups","dt","pain","recovery","anxiety","soreness","dependence","dependency","benzos","cravings","withdraw","withdrawal","addiction"],"range_padding":0.01},"id":"66091","type":"FactorRange"},{"attributes":{},"id":"66097","type":"LinearScale"},{"attributes":{},"id":"66095","type":"CategoricalScale"},{"attributes":{"axis_label":"Recovery Terms","formatter":{"id":"66130","type":"CategoricalTickFormatter"},"major_label_orientation":"vertical","plot":{"id":"66089","subtype":"Figure","type":"Plot"},"ticker":{"id":"66100","type":"CategoricalTicker"}},"id":"66099","type":"CategoricalAxis"},{"attributes":{"bottom_units":"screen","fill_alpha":{"value":0.5},"fill_color":{"value":"lightgrey"},"left_units":"screen","level":"overlay","line_alpha":{"value":1.0},"line_color":{"value":"black"},"line_dash":[4,4],"line_width":{"value":2},"plot":null,"render_mode":"css","right_units":"screen","top_units":"screen"},"id":"66116","type":"BoxAnnotation"},{"attributes":{"active_drag":"auto","active_inspect":"auto","active_multi":null,"active_scroll":"auto","active_tap":"auto","tools":[{"id":"66108","type":"PanTool"},{"id":"66109","type":"WheelZoomTool"},{"id":"66110","type":"BoxZoomTool"},{"id":"66111","type":"SaveTool"},{"id":"66112","type":"ResetTool"},{"id":"66113","type":"HelpTool"}]},"id":"66114","type":"Toolbar"},{"attributes":{},"id":"66113","type":"HelpTool"},{"attributes":{},"id":"66112","type":"ResetTool"},{"attributes":{},"id":"66111","type":"SaveTool"},{"attributes":{"overlay":{"id":"66116","type":"BoxAnnotation"}},"id":"66110","type":"BoxZoomTool"},{"attributes":{},"id":"66130","type":"CategoricalTickFormatter"},{"attributes":{},"id":"66133","type":"Selection"},{"attributes":{},"id":"66132","type":"BasicTickFormatter"},{"attributes":{},"id":"66134","type":"UnionRenderers"},{"attributes":{"below":[{"id":"66099","type":"CategoricalAxis"}],"left":[{"id":"66103","type":"LinearAxis"}],"plot_height":300,"plot_width":2000,"renderers":[{"id":"66099","type":"CategoricalAxis"},{"id":"66102","type":"Grid"},{"id":"66103","type":"LinearAxis"},{"id":"66107","type":"Grid"},{"id":"66116","type":"BoxAnnotation"},{"id":"66126","type":"GlyphRenderer"}],"title":{"id":"66088","type":"Title"},"toolbar":{"id":"66114","type":"Toolbar"},"x_range":{"id":"66091","type":"FactorRange"},"x_scale":{"id":"66095","type":"CategoricalScale"},"y_range":{"id":"66093","type":"DataRange1d"},"y_scale":{"id":"66097","type":"LinearScale"}},"id":"66089","subtype":"Figure","type":"Plot"},{"attributes":{"plot":null,"text":"Recovery Term Counts"},"id":"66088","type":"Title"},{"attributes":{},"id":"66100","type":"CategoricalTicker"},{"attributes":{"grid_line_color":{"value":null},"plot":{"id":"66089","subtype":"Figure","type":"Plot"},"ticker":{"id":"66100","type":"CategoricalTicker"}},"id":"66102","type":"Grid"},{"attributes":{"axis_label":"Number of Drug Users","formatter":{"id":"66132","type":"BasicTickFormatter"},"plot":{"id":"66089","subtype":"Figure","type":"Plot"},"ticker":{"id":"66104","type":"BasicTicker"}},"id":"66103","type":"LinearAxis"},{"attributes":{},"id":"66104","type":"BasicTicker"},{"attributes":{"dimension":1,"plot":{"id":"66089","subtype":"Figure","type":"Plot"},"ticker":{"id":"66104","type":"BasicTicker"}},"id":"66107","type":"Grid"},{"attributes":{},"id":"66108","type":"PanTool"},{"attributes":{},"id":"66109","type":"WheelZoomTool"}],"root_ids":["66089"]},"title":"Bokeh Application","version":"1.0.1"}}
        </script>
        <script type="text/javascript">
          (function() {
            var fn = function() {
              Bokeh.safely(function() {
                (function(root) {
                  function embed_document(root) {
                    
                  var docs_json = document.getElementById('66192').textContent;
                  var render_items = [{"docid":"9ff20b4d-f3f7-446d-87bc-db9dce11ee39","roots":{"66089":"9b1a2c58-0eeb-4f38-bf69-a1cec48bd683"}}];
                  root.Bokeh.embed.embed_items(docs_json, render_items);
                
                  }
                  if (root.Bokeh !== undefined) {
                    embed_document(root);
                  } else {
                    var attempts = 0;
                    var timer = setInterval(function(root) {
                      if (root.Bokeh !== undefined) {
                        embed_document(root);
                        clearInterval(timer);
                      }
                      attempts++;
                      if (attempts > 100) {
                        console.log("Bokeh: ERROR: Unable to run BokehJS code because BokehJS library is missing");
                        clearInterval(timer);
                      }
                    }, 10, root)
                  }
                })(window);
              });
            };
            if (document.readyState != "loading") fn();
            else document.addEventListener("DOMContentLoaded", fn);
          })();
        </script>
    
  </body>
  
</html>