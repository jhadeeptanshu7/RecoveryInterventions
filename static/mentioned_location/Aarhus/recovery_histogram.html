



<!DOCTYPE html>
<html lang="en">
  
  <head>
    
      <meta charset="utf-8">
      <title>Bokeh Plot</title>
      
      
        
          
        <link rel="stylesheet" href="https://cdn.pydata.org/bokeh/release/bokeh-1.0.1.min.css" type="text/css" />
        
        
          
        <script type="text/javascript" src="https://cdn.pydata.org/bokeh/release/bokeh-1.0.1.min.js"></script>
        <script type="text/javascript">
            Bokeh.set_log_level("info");
        </script>
        
      
      
    
  </head>
  
  
  <body>
    
      
        
          
          
            
              <div class="bk-root" id="1664049d-a1bf-4e21-9d62-255616f45d01"></div>
            
          
        
      
      
        <script type="application/json" id="357530">
          {"06b67656-7321-472d-815f-c0d5e0bb0944":{"roots":{"references":[{"attributes":{},"id":"357442","type":"BasicTicker"},{"attributes":{"below":[{"id":"357437","type":"CategoricalAxis"}],"left":[{"id":"357441","type":"LinearAxis"}],"plot_height":300,"plot_width":3500,"renderers":[{"id":"357437","type":"CategoricalAxis"},{"id":"357440","type":"Grid"},{"id":"357441","type":"LinearAxis"},{"id":"357445","type":"Grid"},{"id":"357454","type":"BoxAnnotation"},{"id":"357464","type":"GlyphRenderer"}],"title":{"id":"357426","type":"Title"},"toolbar":{"id":"357452","type":"Toolbar"},"x_range":{"id":"357429","type":"FactorRange"},"x_scale":{"id":"357433","type":"CategoricalScale"},"y_range":{"id":"357431","type":"DataRange1d"},"y_scale":{"id":"357435","type":"LinearScale"}},"id":"357427","subtype":"Figure","type":"Plot"},{"attributes":{"grid_line_color":{"value":null},"plot":{"id":"357427","subtype":"Figure","type":"Plot"},"ticker":{"id":"357438","type":"CategoricalTicker"}},"id":"357440","type":"Grid"},{"attributes":{"fill_color":{"value":"#1f77b4"},"line_color":{"value":"#1f77b4"},"top":{"field":"top"},"width":{"value":0.5},"x":{"field":"x"}},"id":"357462","type":"VBar"},{"attributes":{"callback":null,"factors":["life","anxiety","pain","degree","effects","depression"],"range_padding":0.01},"id":"357429","type":"FactorRange"},{"attributes":{"data_source":{"id":"357461","type":"ColumnDataSource"},"glyph":{"id":"357462","type":"VBar"},"hover_glyph":null,"muted_glyph":null,"nonselection_glyph":{"id":"357463","type":"VBar"},"selection_glyph":null,"view":{"id":"357465","type":"CDSView"}},"id":"357464","type":"GlyphRenderer"},{"attributes":{"callback":null,"data":{"top":[1,1,1,1,1,1],"x":["life","anxiety","pain","degree","effects","depression"]},"selected":{"id":"357471","type":"Selection"},"selection_policy":{"id":"357472","type":"UnionRenderers"}},"id":"357461","type":"ColumnDataSource"},{"attributes":{"fill_alpha":{"value":0.1},"fill_color":{"value":"#1f77b4"},"line_alpha":{"value":0.1},"line_color":{"value":"#1f77b4"},"top":{"field":"top"},"width":{"value":0.5},"x":{"field":"x"}},"id":"357463","type":"VBar"},{"attributes":{"overlay":{"id":"357454","type":"BoxAnnotation"}},"id":"357448","type":"BoxZoomTool"},{"attributes":{},"id":"357447","type":"WheelZoomTool"},{"attributes":{},"id":"357446","type":"PanTool"},{"attributes":{"dimension":1,"plot":{"id":"357427","subtype":"Figure","type":"Plot"},"ticker":{"id":"357442","type":"BasicTicker"}},"id":"357445","type":"Grid"},{"attributes":{},"id":"357468","type":"CategoricalTickFormatter"},{"attributes":{"axis_label":"Number of Drug Users","formatter":{"id":"357470","type":"BasicTickFormatter"},"plot":{"id":"357427","subtype":"Figure","type":"Plot"},"ticker":{"id":"357442","type":"BasicTicker"}},"id":"357441","type":"LinearAxis"},{"attributes":{"plot":null,"text":"Recovery Term Counts"},"id":"357426","type":"Title"},{"attributes":{"source":{"id":"357461","type":"ColumnDataSource"}},"id":"357465","type":"CDSView"},{"attributes":{},"id":"357438","type":"CategoricalTicker"},{"attributes":{"bottom_units":"screen","fill_alpha":{"value":0.5},"fill_color":{"value":"lightgrey"},"left_units":"screen","level":"overlay","line_alpha":{"value":1.0},"line_color":{"value":"black"},"line_dash":[4,4],"line_width":{"value":2},"plot":null,"render_mode":"css","right_units":"screen","top_units":"screen"},"id":"357454","type":"BoxAnnotation"},{"attributes":{},"id":"357450","type":"ResetTool"},{"attributes":{},"id":"357451","type":"HelpTool"},{"attributes":{"active_drag":"auto","active_inspect":"auto","active_multi":null,"active_scroll":"auto","active_tap":"auto","tools":[{"id":"357446","type":"PanTool"},{"id":"357447","type":"WheelZoomTool"},{"id":"357448","type":"BoxZoomTool"},{"id":"357449","type":"SaveTool"},{"id":"357450","type":"ResetTool"},{"id":"357451","type":"HelpTool"}]},"id":"357452","type":"Toolbar"},{"attributes":{},"id":"357472","type":"UnionRenderers"},{"attributes":{},"id":"357449","type":"SaveTool"},{"attributes":{},"id":"357470","type":"BasicTickFormatter"},{"attributes":{},"id":"357471","type":"Selection"},{"attributes":{},"id":"357433","type":"CategoricalScale"},{"attributes":{"callback":null,"start":0},"id":"357431","type":"DataRange1d"},{"attributes":{"axis_label":"Recovery Terms","formatter":{"id":"357468","type":"CategoricalTickFormatter"},"major_label_orientation":"vertical","plot":{"id":"357427","subtype":"Figure","type":"Plot"},"ticker":{"id":"357438","type":"CategoricalTicker"}},"id":"357437","type":"CategoricalAxis"},{"attributes":{},"id":"357435","type":"LinearScale"}],"root_ids":["357427"]},"title":"Bokeh Application","version":"1.0.1"}}
        </script>
        <script type="text/javascript">
          (function() {
            var fn = function() {
              Bokeh.safely(function() {
                (function(root) {
                  function embed_document(root) {
                    
                  var docs_json = document.getElementById('357530').textContent;
                  var render_items = [{"docid":"06b67656-7321-472d-815f-c0d5e0bb0944","roots":{"357427":"1664049d-a1bf-4e21-9d62-255616f45d01"}}];
                  root.Bokeh.embed.embed_items(docs_json, render_items);
                
                  }
                  if (root.Bokeh !== undefined) {
                    embed_document(root);
                  } else {
                    var attempts = 0;
                    var timer = setInterval(function(root) {
                      if (root.Bokeh !== undefined) {
                        embed_document(root);
                        clearInterval(timer);
                      }
                      attempts++;
                      if (attempts > 100) {
                        console.log("Bokeh: ERROR: Unable to run BokehJS code because BokehJS library is missing");
                        clearInterval(timer);
                      }
                    }, 10, root)
                  }
                })(window);
              });
            };
            if (document.readyState != "loading") fn();
            else document.addEventListener("DOMContentLoaded", fn);
          })();
        </script>
    
  </body>
  
</html>