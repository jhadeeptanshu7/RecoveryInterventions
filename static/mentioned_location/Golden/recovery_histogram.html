



<!DOCTYPE html>
<html lang="en">
  
  <head>
    
      <meta charset="utf-8">
      <title>Bokeh Plot</title>
      
      
        
          
        <link rel="stylesheet" href="https://cdn.pydata.org/bokeh/release/bokeh-1.0.1.min.css" type="text/css" />
        
        
          
        <script type="text/javascript" src="https://cdn.pydata.org/bokeh/release/bokeh-1.0.1.min.js"></script>
        <script type="text/javascript">
            Bokeh.set_log_level("info");
        </script>
        
      
      
    
  </head>
  
  
  <body>
    
      
        
          
          
            
              <div class="bk-root" id="3f870659-f8f4-410d-ba43-d64ecdc00a56"></div>
            
          
        
      
      
        <script type="application/json" id="174712">
          {"4cda8026-3243-4e01-b79f-1f1a7042a2c0":{"roots":{"references":[{"attributes":{},"id":"174617","type":"LinearScale"},{"attributes":{},"id":"174615","type":"CategoricalScale"},{"attributes":{"callback":null,"start":0},"id":"174613","type":"DataRange1d"},{"attributes":{"callback":null,"factors":["life","pain","clean","degree","cancer","smart","groups","aa","organization","meetings","death","father","program","treatment","effects","depression","subs","sober","meeting","anxiety","nc","abuse","medical","mother","addiction","personality","alive","quitting","behavior","journey","psychological","church","hospital","territory","cheeks","sobriety","project","disease","anger","court","struggles","hydrated","sickness","programs","benzo","clinic","therapist","meditation","certification","ocd","symptoms","alcoholism","secular","psychiatric","headaches","suffering","therapy","dt","nausea","masters","adulthood","bpd","withdrawal","descent","asleep","sponsor","irritation","er","scalp","teens","avoidance","evaluation","comedown","hungover","withdrawals","behaviour","addictions","intoxicated","benzos","incorporated","urges","hatred","binge","withdraws","medication","foray","withdrawl","seizures","divorce","transform","twenties","withdrawls","frustration","gym","awake","flu","psychosis","rehab","sexuality","cravings","venture","illness","exhausted","inpatient","hospitalization","relapse","na","application","diagnosis","upbringing","phd","discomfort","detox","recovery","nostrils","hungry","jail","psychiatry"],"range_padding":0.01},"id":"174611","type":"FactorRange"},{"attributes":{},"id":"174653","type":"Selection"},{"attributes":{},"id":"174652","type":"BasicTickFormatter"},{"attributes":{},"id":"174650","type":"CategoricalTickFormatter"},{"attributes":{"axis_label":"Recovery Terms","formatter":{"id":"174650","type":"CategoricalTickFormatter"},"major_label_orientation":"vertical","plot":{"id":"174609","subtype":"Figure","type":"Plot"},"ticker":{"id":"174620","type":"CategoricalTicker"}},"id":"174619","type":"CategoricalAxis"},{"attributes":{},"id":"174654","type":"UnionRenderers"},{"attributes":{"active_drag":"auto","active_inspect":"auto","active_multi":null,"active_scroll":"auto","active_tap":"auto","tools":[{"id":"174628","type":"PanTool"},{"id":"174629","type":"WheelZoomTool"},{"id":"174630","type":"BoxZoomTool"},{"id":"174631","type":"SaveTool"},{"id":"174632","type":"ResetTool"},{"id":"174633","type":"HelpTool"}]},"id":"174634","type":"Toolbar"},{"attributes":{"bottom_units":"screen","fill_alpha":{"value":0.5},"fill_color":{"value":"lightgrey"},"left_units":"screen","level":"overlay","line_alpha":{"value":1.0},"line_color":{"value":"black"},"line_dash":[4,4],"line_width":{"value":2},"plot":null,"render_mode":"css","right_units":"screen","top_units":"screen"},"id":"174636","type":"BoxAnnotation"},{"attributes":{},"id":"174631","type":"SaveTool"},{"attributes":{"overlay":{"id":"174636","type":"BoxAnnotation"}},"id":"174630","type":"BoxZoomTool"},{"attributes":{},"id":"174633","type":"HelpTool"},{"attributes":{},"id":"174632","type":"ResetTool"},{"attributes":{"fill_color":{"value":"#1f77b4"},"line_color":{"value":"#1f77b4"},"top":{"field":"top"},"width":{"value":0.2},"x":{"field":"x"}},"id":"174644","type":"VBar"},{"attributes":{"fill_alpha":{"value":0.1},"fill_color":{"value":"#1f77b4"},"line_alpha":{"value":0.1},"line_color":{"value":"#1f77b4"},"top":{"field":"top"},"width":{"value":0.2},"x":{"field":"x"}},"id":"174645","type":"VBar"},{"attributes":{"data_source":{"id":"174643","type":"ColumnDataSource"},"glyph":{"id":"174644","type":"VBar"},"hover_glyph":null,"muted_glyph":null,"nonselection_glyph":{"id":"174645","type":"VBar"},"selection_glyph":null,"view":{"id":"174647","type":"CDSView"}},"id":"174646","type":"GlyphRenderer"},{"attributes":{"source":{"id":"174643","type":"ColumnDataSource"}},"id":"174647","type":"CDSView"},{"attributes":{"callback":null,"data":{"top":[7,6,6,5,5,4,4,4,4,3,3,3,3,3,3,3,3,3,3,3,3,3,3,3,3,2,2,2,2,2,2,2,2,2,2,2,2,2,2,2,2,2,2,2,1,1,1,1,1,1,1,1,1,1,1,1,1,1,1,1,1,1,1,1,1,1,1,1,1,1,1,1,1,1,1,1,1,1,1,1,1,1,1,1,1,1,1,1,1,1,1,1,1,1,1,1,1,1,1,1,1,1,1,1,1,1,1,1,1,1,1,1,1,1,1,1,1,1],"x":["life","pain","clean","degree","cancer","smart","groups","aa","organization","meetings","death","father","program","treatment","effects","depression","subs","sober","meeting","anxiety","nc","abuse","medical","mother","addiction","personality","alive","quitting","behavior","journey","psychological","church","hospital","territory","cheeks","sobriety","project","disease","anger","court","struggles","hydrated","sickness","programs","benzo","clinic","therapist","meditation","certification","ocd","symptoms","alcoholism","secular","psychiatric","headaches","suffering","therapy","dt","nausea","masters","adulthood","bpd","withdrawal","descent","asleep","sponsor","irritation","er","scalp","teens","avoidance","evaluation","comedown","hungover","withdrawals","behaviour","addictions","intoxicated","benzos","incorporated","urges","hatred","binge","withdraws","medication","foray","withdrawl","seizures","divorce","transform","twenties","withdrawls","frustration","gym","awake","flu","psychosis","rehab","sexuality","cravings","venture","illness","exhausted","inpatient","hospitalization","relapse","na","application","diagnosis","upbringing","phd","discomfort","detox","recovery","nostrils","hungry","jail","psychiatry"]},"selected":{"id":"174653","type":"Selection"},"selection_policy":{"id":"174654","type":"UnionRenderers"}},"id":"174643","type":"ColumnDataSource"},{"attributes":{"plot":null,"text":"Recovery Term Counts"},"id":"174608","type":"Title"},{"attributes":{"below":[{"id":"174619","type":"CategoricalAxis"}],"left":[{"id":"174623","type":"LinearAxis"}],"plot_height":300,"plot_width":3500,"renderers":[{"id":"174619","type":"CategoricalAxis"},{"id":"174622","type":"Grid"},{"id":"174623","type":"LinearAxis"},{"id":"174627","type":"Grid"},{"id":"174636","type":"BoxAnnotation"},{"id":"174646","type":"GlyphRenderer"}],"title":{"id":"174608","type":"Title"},"toolbar":{"id":"174634","type":"Toolbar"},"x_range":{"id":"174611","type":"FactorRange"},"x_scale":{"id":"174615","type":"CategoricalScale"},"y_range":{"id":"174613","type":"DataRange1d"},"y_scale":{"id":"174617","type":"LinearScale"}},"id":"174609","subtype":"Figure","type":"Plot"},{"attributes":{},"id":"174628","type":"PanTool"},{"attributes":{},"id":"174629","type":"WheelZoomTool"},{"attributes":{"dimension":1,"plot":{"id":"174609","subtype":"Figure","type":"Plot"},"ticker":{"id":"174624","type":"BasicTicker"}},"id":"174627","type":"Grid"},{"attributes":{},"id":"174624","type":"BasicTicker"},{"attributes":{"grid_line_color":{"value":null},"plot":{"id":"174609","subtype":"Figure","type":"Plot"},"ticker":{"id":"174620","type":"CategoricalTicker"}},"id":"174622","type":"Grid"},{"attributes":{"axis_label":"Number of Drug Users","formatter":{"id":"174652","type":"BasicTickFormatter"},"plot":{"id":"174609","subtype":"Figure","type":"Plot"},"ticker":{"id":"174624","type":"BasicTicker"}},"id":"174623","type":"LinearAxis"},{"attributes":{},"id":"174620","type":"CategoricalTicker"}],"root_ids":["174609"]},"title":"Bokeh Application","version":"1.0.1"}}
        </script>
        <script type="text/javascript">
          (function() {
            var fn = function() {
              Bokeh.safely(function() {
                (function(root) {
                  function embed_document(root) {
                    
                  var docs_json = document.getElementById('174712').textContent;
                  var render_items = [{"docid":"4cda8026-3243-4e01-b79f-1f1a7042a2c0","roots":{"174609":"3f870659-f8f4-410d-ba43-d64ecdc00a56"}}];
                  root.Bokeh.embed.embed_items(docs_json, render_items);
                
                  }
                  if (root.Bokeh !== undefined) {
                    embed_document(root);
                  } else {
                    var attempts = 0;
                    var timer = setInterval(function(root) {
                      if (root.Bokeh !== undefined) {
                        embed_document(root);
                        clearInterval(timer);
                      }
                      attempts++;
                      if (attempts > 100) {
                        console.log("Bokeh: ERROR: Unable to run BokehJS code because BokehJS library is missing");
                        clearInterval(timer);
                      }
                    }, 10, root)
                  }
                })(window);
              });
            };
            if (document.readyState != "loading") fn();
            else document.addEventListener("DOMContentLoaded", fn);
          })();
        </script>
    
  </body>
  
</html>